src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -1663,10 +1663,6 @@ TEST_CASE("SimpleLookupLayerTest", "[ANNLayerTest]")
 
   REQUIRE(outputSum == Approx(arma::accu(output)).epsilon(1e-5));
 
-  // Test the Backward function.
-  module.Backward(input, input, delta);
-  REQUIRE(arma::accu(input) == arma::accu(input));
-
```

Comment:
This test didn't make much sense to me, so I have removed it for now. I think #2398 has the correct test for this.

Reply:
Agreed.